CHANGELOG.md

```diff
@@ -10,6 +10,18 @@ and this project adheres to
 
 + No changes yet
 
+## 14.0.2 - 2020-04-24
+
+### Breaking Change in 14.0.2
```

Comment:
If it's a breaking change, then the major part of the major.minor.patch version number [should](https://semver.org/#spec-item-8) increment, so, 15.0.0.